### ISSUE TITLE
The activation is not quantized!

### ISSUE THREAD

**Issue by viperit** [FLHonker/ZAQ-code#4]
Dear author,

Thank you for the excellent work and resubmission of the code.

However, I found a serious problem which makes your work unreliable at all. As far as I am concerned, the weight is quantized, but the activation is not quantized, which means your method only accomplishes part of the task you mentioned in your paper. This defect makes it hard to follow your paper and I suggest you go over your code and resubmit the new results with your correct code.

If we print the model, there are no "F.relu" modules.
![image](https://user-images.githubusercontent.com/28667932/119075682-c6640f00-ba23-11eb-9be7-4338bd166519.png)

More specially, if we add an indicator in line 80-82 of quantize_model.py, as followed:

![image](https://user-images.githubusercontent.com/28667932/119074057-e9d98a80-ba20-11eb-9ee4-405e0fb676cc.png)

And we run "q_model = quantize_model(q_model, args)", there is no 'ReLU'! That means the activation is not quantized.

To further prove this problem, we use debug mode to stay in line 175 of quant_modules.py which is the Quant_Conv2d's forward function, the position is as followed:
![image](https://user-images.githubusercontent.com/28667932/119074380-813edd80-ba21-11eb-8281-2243e61f996c.png)

We again found the weight is quantized while the activation is not!
As followed:
weight:
![image](https://user-images.githubusercontent.com/28667932/119075750-ed224580-ba23-11eb-9afc-9961dc998289.png)
activation:
![image](https://user-images.githubusercontent.com/28667932/119075783-f6abad80-ba23-11eb-9dfe-8b8bd680da5e.png)

We regard this problem as inconsistency between "nn.ReLU" and "F.relu". The model uses "F.relu" which cannot be recognized by "nn.ReLU" in quantize_model.py. If you use "nn.ReLU" to substitute "F.relu" in model.py, the activation will be quantized in a normal way.

As an example, we conduct a full quantized version of your code which successfully quantizes both the weights and the activations. The quantized results are shown as following,
weight:
![image](https://user-images.githubusercontent.com/28667932/119076212-be589f00-ba24-11eb-983d-6da31f60aefc.png)

activation:
![image](https://user-images.githubusercontent.com/28667932/119076229-c44e8000-ba24-11eb-9a7d-5f6becaae2c1.png)

We think this minimum change is easy to implement and hope you can **_fix the typo and report the new results as soon as possible_**. And we appreciate your work and look forward to cooperating with you in the future.

**Comment by FLHonker** [comment]
Sorry, this repo is part of the code we reconstructed for release and legibility, and some external model.py files are adopted. The bugs may cause you inconvenience. We conducted experiments using the pytorch official models, in which 'nn.ReLU' is employed. We have fix the problems. Thanks for your suggestion!
![image](https://user-images.githubusercontent.com/20468157/119211862-5a4adf00-bae7-11eb-8c0a-a4ba4e7c559e.png)
